lib/airbrake-ruby/response.rb

```diff
@@ -52,5 +63,10 @@ def self.truncated_body(body)
       end
     end
     private_class_method :truncated_body
+
+    def self.rate_limit_delay(response)
+      Time.now + response['X-RateLimit-Delay'].to_i
+    end
```

Comment:
`now + delay` is time when rate limit is reset, not delay

Reply:
i.e. you need to rename method name

Reply:
Renamed to `rate_limit_reset`